### PR TITLE
Add basic subtask info endpoint

### PR DESCRIPTION
**Pull request by bidzyyys**

### CODE REVIEW

File: TranscodingDemo/Server/applogic.py [hunk 1]
```diff
@@ -1,153 +1,180 @@
...
         state_manager = self._get_state_manager(request)
-        state_manager.transition(new_state)     # We should handle somehow failed transitions
+        state_manager.transition(
+            new_state)  # We should handle somehow failed transitions
 
         state = state_manager.state
 
-        state_manager, context = self.functions_map[ state ](request, state_manager)
+        state_manager, context = self.functions_map[state](request,
+                                                           state_manager)
```
Comment (weaselix) [inline TranscodingDemo/Server/applogic.py]: We don't need a limitation to 80 characters per line.

File: TranscodingDemo/Server/rpc.py [hunk 2]
```diff
@@ -1,9 +1,13 @@
 import asyncio
+import logging
 import ssl
 
 import txaio
 from autobahn.asyncio.component import Component
 
+logging.basicConfig(level=logging.INFO)
```
Comment (weaselix) [inline TranscodingDemo/Server/rpc.py]: we configure it in settings.py

File: TranscodingDemo/Server/task.py [hunk 3]
```diff
@@ -134,10 +121,12 @@ def get_allowed_formats(video_params):
...
 def validate(src_params, params):
-
     # Validate resolution
-    if not params["Resolution"] in TranscodingTask.get_allowed_resolutions(src_params):
-        raise ValueError( "Resolution: " + str( src_params["Resolution"] ) + " can't be transcoded to " + str( params["Resolution"] ) )
+    if not params["Resolution"] in TranscodingTask.get_allowed_resolutions(
+            src_params):
+        raise ValueError("Resolution: " + str(
+            src_params["Resolution"]) + " can't be transcoded to " + str(
+            params["Resolution"]))
```
Comment (weaselix) [inline TranscodingDemo/Server/task.py]: Can you instead use `format`?

File: TranscodingDemo/Server/task.py [hunk 4]
```diff
@@ -146,61 +135,36 @@ def validate(src_params, params):
     # Validate codec conversion (including validation of using codec in container)
     allowed_conversions = TranscodingTask.get_allowed_conversions(src_params)
     if not params["VCodec"] in allowed_conversions[params["Container"]]:
-        raise ValueError( "Can't transcode " + src_params["VCodec"] + " to " + params["VCodec"])
+        raise ValueError(
+            "Can't transcode " + src_params["VCodec"] + " to " + params[
+                "VCodec"])
```
Comment (weaselix) [inline TranscodingDemo/Server/task.py]: Can you use format?

File: TranscodingDemo/Server/task.py (hunk 3 above)
Comment (bidzyyys) [reply TranscodingDemo/Server/task.py]: Not my code, but I'll change.

File: TranscodingDemo/Server/task.py (hunk 4 above)
Comment (bidzyyys) [reply TranscodingDemo/Server/task.py]: Not my code, but I'll change.

Comment (weaselix) [reply TranscodingDemo/Server/task.py]: Yes, I see that it is not your code. Thanks for that change.

**Pull request merged**